### ISSUE TITLE
Chrome://accessibility resets upon re-instancing chrome

### ISSUE THREAD

**Issue by Xepheryy** [mmarquee/ui-automation#95]
I noticed that whenever Chrome re-instantiates, the Chrome://accessbility page resets. This severely limits the interaction between the framework and Chrome, my current workaround is to use another browser such as Firefox.

Use case : Web application testing

Is there a suggested way of enabling this access permission in Chrome within the framework?

**Comment by mmarquee** [comment]
You should be able start the browser with `--force-renderer-accessibility` as an option. I've not tested this yet, but I will, and then add it to the documentation, together with an example

See: https://www.chromium.org/developers/design-documents/accessibility

**Comment by Xepheryy** [comment]
In my usage, rather than attaching to the Chrome application after executing it with `--force-renderer-accessibility`/ manually setting it via Chrome://accessibility, I launch the chrome application from my code itself. 
Hence my question in the original comment.

**Comment by mmarquee** [comment]
I have just pushed some changes to the `develop` branch with an example of starting Chrome with the correct parameters. I've added some documentation (using the same example). Can you see whether it works for you (sorry about the _HUGE_ delay, I've been snowed under at work)

**Comment by mmarquee** [comment]
Updated documentation is here ...

https://github.com/mmarquee/ui-automation/blob/master/docs/chrome.md
